tests/Rename-DbaDatabase.Tests.ps1

```diff
@@ -12,8 +12,97 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
         }
     }
 }
-<#
-    Integration test should appear below and are custom to the command you are writing.
-    Read https://github.com/sqlcollaborative/dbatools/blob/development/contributing.md#tests
-    for more guidence.
-#>
\ No newline at end of file
+
+Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        $null = New-DbaDatabase -SqlInstance $script:instance3 -Name 'dbatoolsci_rename1'
```

Comment:
We need to also change this test to be on `$script:instance2`, using `instance3` causes it to run in the larger group of testing.

Reply:
Okay.. I usually write tests against $script:Instance2, and thought I had seen a not about splitting up tests. I will change that as well.. Working on finishing the test this week.